src/runtime/lpython_builtin.py

```diff
@@ -314,3 +314,26 @@ def _lpython_imag(x: c64) -> f64:
 @overload
 def _lpython_imag(x: c32) -> f32:
     return _lfortran_caimag(x)
+
+
+# FloorDiv operator (//)
+
+@overload
+def _lpython_floordiv(a: i32, b: i32) -> i32:
+    return int(a/b)
+
+@overload
+def _lpython_floordiv(a: i32, b: i64) -> i64:
+    return int(a/b)
+
+@overload
+def _lpython_floordiv(a: i64, b: i32) -> i64:
+    return int(a/b)
```

Comment:
I would remove all these. These should be represented in ASR as a division (BinOp) of two integers. The LLVM backend then implements this as integer division and it does what we want. Also the frontend takes care of casting 32 to 64, etc. if needed.